keyboards/arisusplit/readme.md

```diff
@@ -0,0 +1,13 @@
+# arisusplit
+
+A split version of the [Arisu](https://github.com/FateNozomi/arisu-pcb) by FateNozomi, in this handwired version, no split backspace version
```

Comment:
If this is a handwired board per this line, it should be relocated to `keyboards/handwired`